atd-etl/cris_import/.gitignore

```diff
@@ -1 +1,3 @@
-env
```

Comment:
We'll have to remember to delete our existing `env` files since only `.env` is ignored at the top level of the project. I just did mine because I'm paranoid that I'll forget and accidentally leak my 1PW token.

Reply:
ah—thank you. i had the luxury of deleting the old directory and renaming the new directory, so didn't encounter this. i just added `env` to the root `.gitignore` and i will add note to the test steps to remove that. will also drop a note in slack post-go-live